src/share/classes/com/sun/javatest/diff/SuperDiff.java

```diff
@@ -282,8 +282,8 @@ Info getInfo(String path) {
             return infoTable.get(path);
         }
 
-        final Set<String> platforms = new TreeSet<>();
-        final Map<String, Info> infoTable = new HashMap<>();
+        final transient Set<String> platforms = new TreeSet<>();
+        final transient Map<String, Info> infoTable = new HashMap<>();
```

Comment:
This seems totally unrelated

Reply:
I wasn't able to compile jtreg without this modification, and I thought that including this change would help others in similar situation, but I can take it out and either leave it out completely or do a separate PR if you wish?

Reply:
It may be an artifact of the JDK you use to build jtreg.   I recommend using JDK 11.